app/frontend/src/views/Admin.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <v-container>
+    <BaseSecure>
+      <stats />
+    </BaseSecure>
+  </v-container>
+</template>
+
+<script>
+import Stats from '@/components/Stats.vue';
+
+export default {
+  name: 'Admin',
+
+  components: {
+    Stats
+  }
+};
+</script>
```

Comment:
You will get merge conflicts with #150 . You'll have to drop this chunk from your git branch.